### ISSUE TITLE
Fix the feature scaling function

### ISSUE THREAD

**Issue by shobrook** [shobrook/BitVision#60]
I think we're polluting our test set with future data when we standardize it. I need to look into this more and if there's a proper way to scale features for time series prediction tasks.

**Comment by shobrook** [comment]
So, I'm a little stupid and thought we were scaling our features by taking the mean and variance of the whole set (training **and** test), which would introduce future information into the training variables (namely future information contained in the mean and variance).

But we aren't doing that after all... we're fitting our scaler with the training set's mean and variance, and then scaling the test set using only the training mean and variances.

Smfh.
